content/src/content/jcr_root/apps/core/wcm/sandbox/components/tabs/v1/tabs/_cq_dialog/.content.xml

```diff
@@ -55,6 +56,38 @@
                             </columns>
                         </items>
                     </tabitems>
+                    <options
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Options"
```

Comment:
We should be consistent in how we structure / name our dialog tabs. I think we tend to use "Properties" in other components. Also, for the changes to the carousel, I put the properties tab first, and slides tab second but I guess that's open to discussion.

Reply:
I'll update the tab name. I would rather keep this order as I imagine that most often authors will want to edit the items list and not the default one.

Reply:
Wouldn't it make more sense from an authoring POV to have the default tab selection in the same tab as where tabs are defined, just before the multifield?

Reply:
I'm thinking we might need to add some extra options at some point so it makes sense to have a dedicated Properties/Options tab.

Reply:
I agree that we should have a dedicated Properties/Options tab. But it could be empty for now?

Reply:
Renamed the tab in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/317/commits/f1cdc202aed9bbb98c58c7bdd5c5b6ca7a017b4e, let's have a review with @gabrielwalt as well to decide where to put the select.

Reply:
Settled on tabs names in order, updated in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/317/commits/6472388a81a11ad50581f4267d46a757e6a09c9a